src/platform/forms-system/src/js/components/ContactInfoReview.jsx

```diff
@@ -40,61 +43,202 @@ const ContactInfoReview = ({ data, editPage, content, keys }) => {
   );
 
   const dataWrap = data[keys.wrapper] || {};
-  const email = dataWrap[keys.email] || '';
-  const homePhone = dataWrap[keys.homePhone] || {};
-  const mobilePhone = dataWrap[keys.mobilePhone] || {};
-  const address = dataWrap[keys.address] || {};
+  const emailString = dataWrap[keys.email] || '';
+  const homePhoneObj = dataWrap[keys.homePhone] || {};
+  const mobilePhoneObj = dataWrap[keys.mobilePhone] || {};
+  const addressObj = dataWrap[keys.address] || {};
 
-  const isUS = address.addressType !== ADDRESS_TYPES.international;
-  const stateOrProvince = isUS ? 'state' : 'province';
+  const isUS = addressObj.addressType !== ADDRESS_TYPES.international;
 
+  /**
+   * Renders value (if it isn't all whitespace) or an error message wrapped in
+   * a class that makes the text bold & red
+   * @param {String} value - Field value to show
+   * @param {String} errorMessage - Error message text
+   * @returns {String|JSX} - value or error message
+   */
   const showValueOrErrorMessage = (value, errorMessage) =>
-    (value || '').trim() || (
+    (value || '').trim() ||
+    (errorMessage && (
       <span className="usa-input-error-message">{errorMessage}</span>
-    );
+    )) ||
+    '';
 
-  // Label: formatted value in (design) display order
+  /**
+   * Display field label & data (or error message) on the review & submit page
+   * Using an array here to maintain display order
+   * Entry: [ Label, Value or error message ]
+   * - Label = Name of field (customizable in getContent)
+   * - Value or error message = `getValue` function that uses `keys` which is
+   *   the data object key that contains the value; e.g. keys.homePhone matches
+   *   this data path:
+   *   { [wrapperKey]: { [homePhone]: { areaCode: '', phoneNumber: '' } } }
+   * If the value function returns an empty string, the row isn't rendered
+   * */
   const display = [
-    [content.homePhone, () => renderTelephone(homePhone)],
-    [content.mobilePhone, () => renderTelephone(mobilePhone)],
+    [
+      content.homePhone, // label
+      () => {
+        // key.homePhone is undefined if not in `included` option within
+        // `profileContactInfo`
+        if (!keys.homePhone) {
+          return ''; // Don't render row
+        }
+        // content contains the error messages, homePhoneObj is the phone object
+        const errorMsg = validatePhone(content, homePhoneObj);
+        // Pass showValueOrErrorMessage an empty string so error renders
+        return errorMsg
+          ? showValueOrErrorMessage('', errorMsg)
+          : renderTelephone(homePhoneObj); // va-telephone web component
```

Comment:
What happens if `homePhone` is in `included`, but not in `contactInfoRequiredKeys`, and `homePhone` is blank?

Reply:
Right now in 995, the uiSchema validation would block the Veteran from continuing past the contact info page, but if they did manage that, the primary phone page would NOT show and they could submit the form without a home phone